### PR TITLE
Briem Hand: Version 1.002 added

### PR DESCRIPTION
**Pull request by simoncozens**
4c35df2: [gftools-packager] Briem Hand: Version 1.002 added

* Briem Hand Version 1.002 taken from the upstream repo https://github.com/SorkinType/Briem-Hand.git at commit https://github.com/SorkinType/Briem-Hand/commit/.

- [x] Family categorization tags collected from the type design team with the Categories Form
- [ ] Minisite_url definition in the METADATA.pb file for commissioned projects
- ~~[ ] Primary_script definition in the METADATA.pb file for all projects that have a primary non-Latin based language support target~~
- [x] Fontbakery checks are reviewed and failing checks are resolved in collaboration with the upstream font development team
- ~~[ ] Diffenator2 regression checks for revisions on all projects that are currently in production~~
- [ ] Designers bio info have to be present in the designer catalog (at least an issue should be opened for tracking this, if they are not) https://github.com/google/fonts/pull/7475
- [x] Check designers order in metadata.pb, since the first one of the list appears as “principal designer”
- [ ] Social media formatted visual assets for all new commissioned projects in the Drive directory, communicate with the repository Maintainer so that they can push this content to the Social Media tracker spreadsheet
- [ ] Social media content draft for all new commissioned projects in the Drive directory and Social Media tracker spreadsheet, communicate with the repository Maintainer so that they can push this content to the Social Media tracker spreadsheet

### CODE REVIEW

**Comment by emmamarichal** [comment]
Hi @simoncozens, do you need I review it or you're still working on it?

**Comment by simoncozens** [comment]
Upstream is working on it. :-) I'll try and fix as many of these as I can, some are from the new build process.

**Comment by emmamarichal** [comment]
- For the font name: I already had this kind of issue, with Darker Grotesque for example: https://fonts.google.com/specimen/Darker+Grotesque  I didn't change the name (the style displayed on the API is the regular), and I ignored the fail. But we can discuss this with the team!
- I can add Gunnlaugur Briem to the catalog
- ⚠ WARN: DESCRIPTION.en_us.html should end in a linebreak. -> Can be avoid, I can do it when all is finished
- ⚠ WARN: Does the font contain a soft hyphen?  Soft Hyphen can be removed from the Glyphset

You can fix the interpolation bugs, here are what I found when I inspected quickly on font gauntlet:

The ogonek should be shifted a bit
Capture d’écran 2023-11-30 à 14 49 44

Interpolation issue here (it's maybe already on the list, I didn't check)

https://github.com/google/fonts/assets/64773544/ea840ae6-c3f1-404f-bebf-96ef3081b7ae

The ring seems placed too much on the left

https://github.com/google/fonts/assets/64773544/705fc784-4dc6-4540-abaf-7fd5ccb52a00

**Comment by simoncozens** [comment]
Interpolation issues:

- [x] downArrow (false positive)
- [x] o.c (false positive, or at least not a visible issue)
- [x] c.a (fixed)
- [x] r.s (fixed)
- [x] section (false positive)
- [x] upArrow (false positive)
- [x] v.c (Fixed)
- [x] z.c (fixed)
- [x] r.c (fixed)
- [ ] o.v
- [x] rtail (fixed)
- [ ] o.z

I think `o.v` and `o.z` are fine but want to confirm with Eben:

Screenshot 2023-11-30 at 15 16 29

**Comment by emmamarichal** [comment]
Hey @simoncozens!

Looks great! 🥳 

Just a few things:

- the A ring is still too far to the left

Capture d’écran 2023-12-01 à 10 10 59

- The lcaron kerning with 'k' /  and lbar with the 'w'
I don't have any solution, and I imagine it's hard to deal with this kind of things with this design. 
cc @EbenSorkin 

Capture d’écran 2023-12-01 à 10 23 20

**Comment by simoncozens** [comment]
Well spotted. I can fix the accents on Aringbelow, kerning will have to be a designer job.

**Comment by RosaWagner** [comment]
About this FAIL:
🔥 FAIL: METADATA.pb font.name field contains font name in right format? ([com.google.fonts/check/metadata/valid_name_values](https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/metadata/valid_name_values))
🔥 FAIL METADATA.pb font.name field ("Briem Hand") does not match correct font name format ("Briem Hand Thin"). [code: mismatch]

It is a bug from fontbakery, issue opened here: https://github.com/fonttools/fontbakery/issues/4262. The `font_name` key should be matching the `family name`, not `family name + style name`.

**Comment by simoncozens** [comment]
Uh, Noto Sans Mongolian fix clearly unrelated. Naughty packager. Let me force-push...

File: ofl/briemhand/METADATA.pb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  files {
+    source_file: "DESCRIPTION.en_us.html"
+    dest_file: "DESCRIPTION.en_us.html"
+  }
+  files {
+    source_file: "fonts/variable/BriemHand[wght].ttf"
+    dest_file: "BriemHand[wght].ttf"
+  }
+  branch: "main"
+}
```
Comment (emmamarichal) [inline ofl/briemhand/METADATA.pb:43]: should we add `classifications: "HANDWRITING"`?

File: ofl/briemhand/OFL.txt [hunk 2]
```diff
@@ -0,0 +1,93 @@
+Copyright 2023 The Briem Hand Project Authors (https://github.com/SorkinType/Briem-Hand)
+
+This Font Software is licensed under the SIL Open Font License, Version 1.1.
+This license is copied below, and is also available with a FAQ at:
+https://scripts.sil.org/OFL
```
Comment (emmamarichal) [inline ofl/briemhand/OFL.txt:5]: We should use the new link: `https://openfontlicense.org`

**Review by emmamarichal — CHANGES_REQUESTED** [review]

**Comment by emmamarichal** [comment]
@simoncozens The fontbakery looks good to me.


### Design review:

- There are still some interpolation issues, but I don't think it's visible. But maybe better to fix if you can!
[BriemHand[wght].pdf](https://github.com/google/fonts/files/14773534/BriemHand.wght.pdf)

- This guy (dong) is a bit too far:

Screenshot 2024-03-27 at 13 43 50

https://github.com/google/fonts/assets/64773544/b575af71-159d-4c26-97f9-2807efac17d4

- Small issue here in the outlines (rupeeIndian):


Screenshot 2024-03-27 at 13 44 15

**Comment by emmamarichal** [comment]
@simoncozens Yes, that's what I thought about the interpolation problems.
But I was thinking that if you fixed the other little problems I mentioned, you could fix the interpolations to avoid a WARN hehe
But it doesn't matter if you don't!

**Comment by simoncozens** [comment]
I've fixed the dong issue in the repository, I don't feel confident fixing the rupee outlines so that's over to @EbenSorkin.

**Comment by EbenSorkin** [comment]
I'll see if I can fix this today.

File: ofl/briemhand/METADATA.pb [hunk 3]
```diff
@@ -0,0 +1,43 @@
+name: "Briem Hand"
+designer: "Gunnlaugur Briem, Eben Sorkin"
```
Comment (emmamarichal) [inline ofl/briemhand/METADATA.pb:2]: you can add SE to get Gunnlaugur SE Briem

Comment (emmamarichal) [reply ofl/briemhand/METADATA.pb:2]: I've updated the designer profile

**Comment by simoncozens** [comment]
Updated

**Comment by simoncozens** [comment]
OK, so it was probably a bad time to adjust the axis values... ;-)

**Comment by simoncozens** [comment]
Updated

**Comment by emmamarichal** [comment]
@simoncozens 

I have this in the last report: `💥 **ERROR** Familyname must be unique according to namecheck.fontdata.com`
I just updated metadatas and OFL. Any idea? A fontbakery bug?

**Comment by m4rc1e** [comment]
I think we're making too many requests to namecheck.fontdata.com and we've potentially crashed their server so we should ignore it.

We should be good citizens and only run the check on new families so we need to implement a cache in gb

**Review by emmamarichal — APPROVED** [review]

**Pull request merged**